scripts/assessment-cof-reporting.sql

```diff
@@ -0,0 +1,24 @@
+SELECT
+	application_id,
+	short_id,
+	jsonb_path_query_first (jsonb_blob, '$.forms[*].questions[*].fields[*] ? (@.key == "WWWWxy")."answer"') AS "Unique Tracker Number",
+	jsonb_path_query_first (jsonb_blob, '$.forms[*].questions[*].fields[*] ? (@.key == "YdtlQZ")."answer"') AS "Organisation Name",
+	jsonb_path_query_first (jsonb_blob, '$.forms[*].questions[*].fields[*] ? (@.key == "lajFtB")."answer"') AS "Organisation Type",
+	jsonb_path_query_first (jsonb_blob, '$.forms[*].questions[*].fields[*] ? (@.key == "KAgrBz")."answer"') AS "Project Name",
+	jsonb_path_query_first (jsonb_blob, '$.forms[*].questions[*].fields[*] ? (@.key == "LZBrEj")."answer"') AS "Lead Contact Email Address",
+	funding_amount_requested AS "Funding Amount Requested",
+	jsonb_path_query_first (jsonb_blob, '$.forms[*].questions[*].fields[*] ? (@.key == "NdFwgy")."answer"') AS "Capital Costs",
+	jsonb_path_query_first (jsonb_blob, '$.forms[*].questions[*].fields[*] ? (@.key == "NyudvF")."answer"') AS "Revenue Costs",
+	asset_type AS "Asset Type",
+	jsonb_path_query_first (jsonb_blob, '$.forms[*].questions[*].fields[*] ? (@.key == "GjzaqR")."answer"') AS "Asset Type Other",
+
+	jsonb_path_query_first (jsonb_blob, '$.forms[*].questions[*].fields[*] ? (@.key == "yEmHpp")."answer"') AS "Address",
+
+	location_json_blob -> 'postcode' AS "Postcode",
+	location_json_blob -> 'county' AS "County",
+	location_json_blob -> 'country' AS "Country",
+	location_json_blob -> 'constituency' AS "Constituency"
```

Comment:
This doesn't include any scores or comments, are we planning on adding those or doing them in separate queries?

Reply:
Can we convert to a draft PR if we're still amending it?

Reply:
Scores and comments will be pulled in as separate queries but these are simple queries. I wasn't planning on adding them to github as they're so simple however

Reply:
If there is some sort of manual recipe we're using to pull the final CSV together, can we still add these queries and document the process as-is? Ideally we should be able to pull all the data in a single query but if not at least document the process as-is.

Reply:
Other scripts included. Once I retrieve the data I'll merge it together as requested by Martin manually